src/deposit_event.rs

```diff
@@ -62,4 +61,20 @@ impl DepositedEvent {
             fee,
         }
     }
+
+    #[cfg(not(feature = "contract"))]
+    #[allow(dead_code)]
+    pub fn to_log_entry_data(&self) -> Vec<u8> {
+        EthEvent::to_log_entry_data(
+            DEPOSITED_EVENT,
+            DepositedEvent::event_params(),
+            self.eth_custodian_address,
+            vec![self.sender.to_vec()],
+            vec![
+                ethabi::Token::String(self.recipient.clone()),
+                ethabi::Token::Uint(self.amount.into()),
+                ethabi::Token::Uint(self.fee.clone()),
+            ],
+        )
+    }
```

Comment:
Any reason to have this? I don't see it's used anywhere. Probably should be removed along with `EthEvent::to_log_entry_data`.

Reply:
It was useful for me to craft data for testing. Let's keep this here.

I think most of the tests that uses Proofs as very long string objects (that contains a json) should be rewritten creating the actual objects and serializing later. The main reason for this, is that if a test starts failing, debugging is a nightmare for those unfamiliar with this test, given that this data is almost a black box.

Reply:
Got it, then sure, let's keep this!